code/drasil-printers/lib/Drasil/Database/SearchTools.hs

```diff
@@ -0,0 +1,77 @@
+module Drasil.Database.SearchTools where
```

Comment:
A grab-bag of tools for finding chunks from the ChunkDB. I do not want them in `drasil-database` because I think that `drasil-database` should be a standalone package (because the implementation is now mostly independant of any specific chunks).